devel/CLI11/Portfile

```diff
@@ -1,24 +1,43 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           github 1.0
```

Comment:
Is the re-ordering necessary or a common practice?

Reply:
> Is the re-ordering necessary or a common practice?

Alphabetic order is a recommended practice, but not obligatory. If you object to that, no issues, I will drop this part.

Reply:
Sure, gonna drop it now.

Reply:
Let me just drop non-functional commit completely in fact.